### PR TITLE
Add content guidelines and other tweaks to community page

### PR DESCRIPTION
**Pull request by kriswest**
This PR: 
- adds content guidelines to the community data file, 
- adds a content disclaimer to the community page 
- adjusts the community page filter categories and recategorizes existing entries
  - "Examples and training" was split into "Tools and training" (training courses, software tools, conformance tests) and "Demos" (app and platform demos, getting started tools).
  - An "other" category was created for everything else (meetups, surveys, youtube videos etc.)
  - Adjusted margins on filter buttons (which became a bit spread out during docusaurus 2 update)
- adds a few missing "Open Source" badges
- stops the badges being clickable (just resets the scroll by reloading the page)
- adopters moved become first and default filter,
- removed the all filter,
- ensured that a filter is always selected,
- switched to a proper react implementation of filter to resolve oddities.

### CODE REVIEW

**Comment by kriswest** [comment]
@nkolba @robmoffat @finos/fdc3-maintainers Here's a crack at the discussed tweaks to the community page. I figured the easiest place to put the content guidelines is in the JSON file as that's what you see when you click on the "add your details" link.

If this is good with you, then we need to merge it and just lightly tweak Nick's PR to set the right categories, etc.

Happy to take suggestions on the guidelines, disclaimer etc., this is just an attempt to move us forward ;-) 

Preview link: https://deploy-preview-1169--fdc3.netlify.app/community

**Comment by kriswest** [comment]
P.S. Implementation filters need rewriting in react at some point. The community page is currently rendered by a react functional component then hacked by some old JS I wrote for the static version in the docusaurus 2 site. There should be some state for the selected filter, which is then used to filter the rendered entries and highlight the selected filter.

The main downside currently is that the default filter is not shown as selected. Its possible it will develop other display artefacts later.

**Review by bingenito — APPROVED** [review]
I like this

File: website/data/community.json [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+    "//badges1": "Open Source: Indicates that the project source code is available to download and modify, under an Apache 2.0 or similar license.",
+    "//badges2": "FDC3 1.2 Supported: Indicates that this product advertises compatibility with the FDC3 1.2 Standard. ",
+    "//badges3": "FDC3 2.0 Supported: Indicates that this product advertises compatibility with the FDC3 2.0 Standard. ",
+    "//badges4": "FDC3 1.2 Compliant: This badge is applied to desktop agents that have passed the FINOS FDC3 1.2 Conformance testing process.",
+    "//badges5": "FDC3 2.0 Compliant: This badge is applied to desktop agents that have passed the FINOS FDC3 2.0 Conformance testing process.",
+    "//conformance": "Indicate that a desktop agent has passed a FINOS FDC3 Conformance testing process - see other entries for examples of how to set this.",
+    "//description": "A description to be displayed by your entry. Please limit to 100 hundred words and avoid the use of superlatives (e.g. easiest, best, most powerful etc.) or other descriptive content that may be disputed by other community members."
```
Comment (kriswest) [inline website/data/community.json:17]:
```suggestion
    "//description": "A description to be displayed by your entry. Please limit to 100 words and avoid the use of superlatives (e.g. easiest, best, most powerful etc.) or other descriptive content that may be disputed by other community members."
```

**Review by bingenito — APPROVED** [review]

**Comment by kriswest** [comment]
From the last maintainers meeting:

- Community page PR and content guidelines
  - All ok with content guidelines etc.
  - Still a problem with multiple entries clustering by alpha sort
    - Consider selecting a category/eliminating all filter
    - Move adopters to be first category
    - Encourage more adopter entries - we've got many super involved firms not listed: Citi, Morgan Stanley, RBC, Wellington etc., could add big name app providers too (if multi-entry problem solved) e.g. Factset and Symphony. The goal of this whole page and specifically the adopters section is to show engagement with FDC3 so its where we need to focus efforts
      - Consider sending out proposed entries, odds are they will correct and send back faster than we get them to originate one...

Actions:
- (Kris) Take community PR a bit further if possible - adopters to become first section, remove all filter, ensure a filter is selected by default, switch to proper react implementation of filter to resolve oddities.

**Review by hughtroeger — APPROVED** [review]

**Comment by kriswest** [comment]
@nkolba This is merged now, so if you could merge into your PR and check the content categories that should be good to go too

**Pull request merged**